src/assets/styles/components/_mms-search.scss

```diff
@@ -26,6 +26,13 @@
       padding-bottom: 0;
     }
 
+    .close-button-container {
```

Comment:
this is really nice to have but when you have the right pane open it looks strange... i'm going to submit a change

Reply:
it is fixed now.